style.css

```diff
@@ -17,6 +12,134 @@ body {
   align-items: center;
 }
 
+@media screen and (min-width: 768px) {
+  body {
+    width: auto;
+    margin: 0 auto;
+    margin-left: 40px;
+    margin-right: 40px;
+  }
```

Comment:
- [ ] For this milestone, the recommended break-point for desktop screen size is between 992px - 1100px. Change your break-point.

Reply:
Thank you.